src/modules/esl-utils/misc/test/format.test.ts

```diff
@@ -112,4 +114,35 @@ describe('misc/format helper tests', () => {
       (args, exp) => expect(parseNumber.apply(null, args)).toBe(exp)
     );
   });
+
+  describe('parseTime', () => {
+    test.each([
+      [['.3s'], 300],
+      [['.124s'], 124],
+      [['4s'], 4000],
+      [['350ms'], 350],
+      [['1024ms'], 1024],
```

Comment:
I suggest adding test cases for these values:
4.s
350.ms
Look like these is invalid values.